### PR TITLE
Adds the Honk Render as a Traitor Item for clowns

### PR DESCRIPTION
**Pull request by TheOneAndOnlyCreeperJoe**
Takes the unused Honk Render and gives it to Traitor Clowns. Costs 7 so you can only buy 2 of these, which considering it lets you spawn 10 clowns each is pretty balanced.

Blade itself deals 15 brute damage.

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Can you add the honk noise to hit_sound?

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Done

**Comment by Crystalwarrior** [comment]
Mmmm yes this is Gucci

File: code/game/gamemodes/wizard/artefact.dm [hunk 1]
```diff
@@ -88,6 +88,10 @@
 	activate_descriptor = "depression"
 	rend_desc = "Gently wafting with the sounds of endless laughter."
 	icon_state = "clownrender"
+/obj/item/weapon/veilrender/honkrender/attack(mob/living/carbon/M, mob/living/carbon/user)
+	var/honknoise = 'sound/items/bikehorn.ogg'
```
Comment (Carbonhell) [inline code/game/gamemodes/wizard/artefact.dm]: um..you could just do playsound(loc, 'sound/items/bikehorn.ogg', 50, 1, -1)

Comment (TheOneAndOnlyCreeperJoe) [inline code/game/gamemodes/wizard/artefact.dm]: Making it a var allows adminbus to modify the sound.

Comment (TDcoolguy300) [inline code/game/gamemodes/wizard/artefact.dm]: You should make it a variable on the base render then, not just the Clown one.

Comment (TDcoolguy300) [inline code/game/gamemodes/wizard/artefact.dm]: Also, it wouldnt work within the attack proc, it has to be defined with the description and everything else

**Comment by Nebyab** [comment]
ok

**Comment by dreamcrab** [comment]
sure why not

**Comment by Carbonhell** [comment]
Code's ok, merging

**Pull request merged**